src/controllers/organizations.js

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {
+  createResponse,
+  badRequest,
+  noContent,
+  notFound,
+  ok,
+} from '@adobe/spacecat-shared-http-utils';
+import {
+  hasText,
+  isObject,
+  isString,
+} from '@adobe/spacecat-shared-utils';
+import { createOrganization as validateOrganization } from '@adobe/spacecat-shared-data-access/src/models/organization.js';
+
+import { OrganizationDto } from '../dto/organization.js';
+import { SiteDto } from '../dto/site.js';
+
+/**
+ * Organizations controller. Provides methods to create, read, update and delete organizations.
+ * @param {DataAccess} dataAccess - Data access.
+ * @returns {object} Organizations controller.
+ * @constructor
+ */
+function OrganizationsController(dataAccess) {
+  if (!isObject(dataAccess)) {
+    throw new Error('Data access required');
+  }
+
+  /**
+   * Creates an organization. The organization ID is generated automatically.
+   * @param {object} context - Context of the request.
+   * @return {Promise<Response>} Organization response.
+   */
+  const createOrganization = async (context) => {
+    try {
+      validateOrganization(context.data);
+      const organization = await dataAccess.addOrganization(context.data);
+      return createResponse(OrganizationDto.toJSON(organization), 201);
```

Comment:
we should add `created(data)` to `@adobe/spacecat-shared-http-utils` just like `ok`, `notFound` to be consistent